public/assets/css/icons/rss.css

```diff
@@ -0,0 +1,47 @@
+.icon .rss {
+    width: 32px;
+    height: 33px;
+    background: #ea4335;    
+}
+
+.rss .icon-part-one {
+    width: 20px;
+    height: 20px;
+    border-radius: 0 0 100%;    
+    background: white;
+    transform:translate3d(40%, 20%, 0) rotate(-90deg);
```

Comment:
rotate it's not necessary, it's better to use this code with a target
you can also use this:
```css           
border-top-right-radius: 100%;                                                                      
```
[exemple playground](https://jsfiddle.net/0ubv6wds/)

Reply:
we can talk about this one together, if iapply it everything must be changed (=position) and the code is heavier